### DISCUSSION TITLE
Need to create a new instance of Seq on Azure, better to use a VM or docker

### DISCUSSION THREAD

**Discussion by advapiIT** [datalust/seq-tickets#1724] [Help]
Hello,
For an existing application that currently doesn't save logs in a really good way, I decided to integrate Seq and I was wondering if It's better to have a VM for that or to use a Docker + Blob storage to store data... what's the best approach? in the case of a VM , which are the min requirements?
thanks

**Comment by liammclennan** [comment]
Hi @advapiIT ,

Seq can be deployed in a VM or as a docker image (see https://docs.datalust.co for all the options). 

Hardware requirements depend upon load, but here are some guidelines (from [System Requirements](https://docs.datalust.co/docs/system-requirements)):

Seq requires a 64-bit physical or virtual machine.

For most light workloads - a few events per second, retained 30 days, - a basic machine with a minimum of the following specifications will provide very good performance:

    2 CPU cores
    3 GB RAM
    50 GB storage

For busier servers, or where many developers use Seq concurrently, additional CPU, RAM and storage will be required. As a rule of thumb, 1 GB RAM per 1.5 million retained events is a good starting point. CPU and IO requirements are highly dependent on the nature of the events being logged, so measuring and tuning based on usage will be necessary.

A starting configuration for a loaded production server is:

    4 CPU cores
    16 GB RAM
    256 GB SSD storage

    📘

    Seq performs best with SSD-backed storage.

Seq runs in a single-node configuration, so scaling up or down requires adjustment of these numbers, rather than the addition of machines.
